### PR TITLE
kOps: Add pre-submit for AWS clusters with loadbalancer

### PR DESCRIPTION
**Pull request by hakman**
/cc @olemarkus @johngmyers @rifelpet

### CODE REVIEW

**Review by zetaab — APPROVED** [review]
/lgtm

File: config/jobs/kubernetes/kops/build_jobs.py [hunk 1]
```diff
@@ -1171,6 +1171,17 @@ def generate_presubmits_e2e():
             extra_flags=["--dns=none"],
         ),
 
+        presubmit_test(
+            name="pull-kops-e2e-aws-nlb",
+            cloud="aws",
+            distro="u2204",
+            networking="calico",
+            extra_flags=[
+                "--api-loadbalancer-type=public"
```
Comment (johngmyers) [inline config/jobs/kubernetes/kops/build_jobs.py:1180]: Missing trailing comma.

File: config/jobs/kubernetes/kops/kops-presubmits-e2e.yaml [hunk 2]
```diff
@@ -1142,6 +1142,71 @@ presubmits:
       testgrid-days-of-results: '90'
       testgrid-tab-name: pull-kops-e2e-aws-dns-none
 
+# {"cloud": "aws", "container_runtime": "containerd", "distro": "u2204", "extra_flags": "--api-loadbalancer-type=public --discovery-store=s3://k8s-kops-prow/discovery", "k8s_version": "stable", "kops_channel": "alpha", "networking": "calico"}
+  - name: pull-kops-e2e-aws-lb
```
Comment (johngmyers) [inline config/jobs/kubernetes/kops/kops-presubmits-e2e.yaml:1146]: How does this end "-lb" when build_jobs.py has the name end with "-nlb"?

When I run build_jobs.py, I get uncommitted changes.

File: config/jobs/kubernetes/kops/build_jobs.py (hunk 1 above)
Comment (hakman) [reply config/jobs/kubernetes/kops/build_jobs.py:1180]: Fixed in https://github.com/kubernetes/test-infra/pull/27839

File: config/jobs/kubernetes/kops/kops-presubmits-e2e.yaml (hunk 2 above)
Comment (hakman) [reply config/jobs/kubernetes/kops/kops-presubmits-e2e.yaml:1146]: Fixed in https://github.com/kubernetes/test-infra/pull/27839.

**Pull request merged**